classes/locations.php

```diff
@@ -63,7 +69,7 @@ public function register_locations() {
 			'register_meta_box_cb' => array( &$this, 'location_meta_cb' ),
 			'supports'             => array( 'title','author', 'editor'),
 			'labels'               => array(
-				'name'               => 'SimpleMap',
+				'name'               => 'SimpleMap Locations',
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed

<!-- Comment by [SideCI](https://sideci.com) -->